elasticsearch/src/main/java/org/hibernate/search/elasticsearch/cfg/ElasticsearchEnvironment.java

```diff
@@ -18,6 +18,7 @@
 	 */
 	public static final class Defaults {
 
+		public static final String DYNAMIC_MAPPING = "strict";
```

Comment:
@DavideD Caution with the existing enum: it's part of an implementation package, and for a good reason. It's used for building our internal representation of the Elasticsearch mapping, and as such it shouldn't be exposed.

We can always add another enum, if it's really necessary.